.setup/bin/setup_sample_courses.py

```diff
@@ -443,6 +446,185 @@ def create_gradeable_submission(src, dst):
         os.remove(zip_dst)
 
 
+def make_sample_json():
```

Comment:
So we're not always guaranteed to be building the sample course (if we only pass in say tutorial or whatever), I'd rather move the "make_sample_json()" function inside the course loop (right after the create function?) and make it a configuration option in the yml file (false by default).